section2/exercises/comparisons.js

```diff
@@ -110,22 +114,45 @@ var age = 1;
 
 // YOU DO:
 // Determine if the dog loves to play and loves treats
-
+console.log(lovesToPlay && lovesTreats); // true
 
 // Determine if the dog loves to play and loves the dog park
-
+console.log(lovesToPlay && lovesDogPark); // false
 
 // Determine if the dog loves to play or loves the dog park
-
+console.log(lovesToPlay || lovesDogPark); // true
 
 // Determine if the dog loves to play and is a puppy
+console.log(lovesToPlay && age); // 1
```

Comment:
🚨Hmmm... this should return a `true` or `false`
Hint: how could you modify the `age` section, or truly, what conditions could you place on `age` to help you out here?